src/main/java/com/google/sps/data/InsertAndUpdate.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.Post;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InsertAndUpdate{
```

Comment:
I had talked to Swetha about a better way to structure and name these classes (InsertAndUpdate, Parse, and Query). Is she going to address that in another PR. If so, please leave a TODO.